docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -440,6 +440,31 @@ return Person.find("status", Status.Alive)
 
 The `PanacheQuery` type has many other methods to deal with paging and returning streams.
 
+=== Using a range instead of pages
+
+`PanacheQuery` also allow range based query instead of page based.
+
+[source,java]
+----
+// create a query for all living persons
+PanacheQuery<Person> livingPersons = Person.find("status", Status.Alive);
+
+// make it use a range or start index 0 and last index 25.
+livingPersons.range(0, 25);
+
+// get the range
+List<Person> firstRange = livingPersons.list();
+
+// to get the next range, you need to call range again
```

Comment:
Oh hold on, I thought the range was inclusive on the last index? If it is, then you get element 25 for the previous range and this one too, no?

Reply:
Yes, the example should be `range(0, 24)` then `range(25, 49)` to have two range of 25 items as range starts at 0.
I'll update the documentation.

We can decide to make it start at 1 if needed, but in IT stuff starts at 0 ;)

Reply:
OK fine.